docs/Guides/Sell/FindSKU.md

```diff
@@ -0,0 +1,65 @@
+# Find a SKU
+
+We currently only support creating orders for products that don't include order forms. An API to list the SKUs from your store will be comming. In the mean time please use this list or reachout to support if you can't find a product you are looking for.
+
+The SKU is typically different between the demo and production environments so be sure to use the correct values.
+
+## Your Packages
+You can use packages that you have built in your store so long as none of the products require order forms. If products are added or removed from the package at a later date future orders will reflect the change.
+
+Currently the best way to get the SKU for a package is to go to **Partner Center -> Marketplace -> Manage Store** and select your package. The page URL will contain the SKU for the package.
+
+Package SKUs start with `SOL-`. Example: `SOL-b4ee54bd-ea26-4047-b3e7-4ba075b6d1d3`
```

Comment:
Based on [this thread](https://vendasta.slack.com/archives/C2LTW227R/p1619718933078300) we may want to remove any mention of packages for now.